### PR TITLE
TigerOS Website

### PR DESCRIPTION
**Pull request by jrtechs**
Initial creation of the TIgerOS website.

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -1 +1,322 @@
...
+              <a href="https://github.com/RITlug/TigerOS" class="btn btn-primary">Find our project</a>
+            </div>
+          </div>
+        </div>
+        <div class="col-sm-4 my-4">
+          <div class="card">
+            <img class="card-img-top" src="https://opensource.org/files/osi_keyhole_300X300_90ppi_0.png" alt="">
+            <div class="card-body">
+              <h4 class="card-title">Open Source</h4>
+              <p class="card-text">TigerOS is a fully open source project, meaning the source code is public for redistribution and contribution.</p>
```
Comment (axk4545) [inline docs/index.html]: Maybe "publicly available for use and modification"

File: docs/index.html [hunk 2]
```diff
@@ -1 +1,322 @@
...
+              <a href="http://ritlug.com/" class="btn btn-primary">RITLUG</a>
+            </div>
+          </div>
+        </div>
+        <div class="col-sm-4 my-4">
+          <div class="card">
+            <img class="card-img-top" src="img/Octocat.png" alt="">
+            <div class="card-body">
+              <h4 class="card-title">Github</h4>
+              <p class="card-text">Github is an online colaborative website where we hoste the code for Tiger-OS.</p>
```
Comment (axk4545) [inline docs/index.html]: Please fix spelling. Also feel free to pull the description for this directly from GitHub itself

File: docs/index.html [hunk 3]
```diff
@@ -1 +1,322 @@
...
+          <h2 class="mt-4">What is TigerOS?</h2>
+          <p>A RIT student developed, Fedora based operating system that beautifully utilizes linux to meet the demands of RIT sutdents and faculty.</p>
+          <p>TigerOS is maintained and developed by the Rochester Intitute of Technology's Linux User Group(<a href="http://ritlug.com/">ritlug</a>).</p>
+          <p>
+            <a class="btn btn-primary btn-lg" href="#">Learn more &raquo;</a>
+          </p>
+        </div>
+        <div class="col-sm-4">
+          <h2 class="mt-4">Download TigerOS</h2>
+          <p>A fast and open source alternative to windows and mac.</p>
```
Comment (axk4545) [inline docs/index.html]: @jflory7 sounds kinda odd. suggestions?

Comment (axk4545) [reply docs/index.html]: maybe "a fast and free alternative to macOS and Windows" ?

File: docs/index.html [hunk 4]
```diff
@@ -1 +1,322 @@
...
+      </div>
+    </header>
+
+    <!-- Page Content -->
+    <div class="container">
+
+      <div class="row">
+        <div class="col-sm-8">
+          <h2 class="mt-4">What is TigerOS?</h2>
+          <p>A RIT student developed, Fedora based operating system that beautifully utilizes linux to meet the demands of RIT sutdents and faculty.</p>
```
Comment (axk4545) [inline docs/index.html]: @jflory7 suggestions on this wording?

File: docs/index.html [hunk 5]
```diff
@@ -1 +1,322 @@
...
+          </p>
+        </div>
+      </div>
+      <!-- /.row -->
+
+      <!-- image slider -->
+      <div class="row">
+        <div class="col-sm-8">
+          <h2 class="mt-4">Screenshots</h2>
+          <p>Here are a couple screenshots of TigerOS to give you a feel for the OS.</p>
```
Comment (axk4545) [inline docs/index.html]: remove that line. seems unneccessary

File: docs/index.html (hunk 3 above)
Comment (Tjzabel) [reply docs/index.html]: @axk4545 I like how you word it. "A fast and free alternative to macOS and Windows."

File: docs/index.html (hunk 5 above)
Comment (Tjzabel) [reply docs/index.html]: I agree.

File: docs/index.html (hunk 4 above)
Comment (ct-martin) [reply docs/index.html]: Should be "Fedora Remix" b/c legal

**Comment by axk4545** [comment]
leaving for @ct-martin @jrtechs to work through during Friday hacksesh also so @jflory7 can review,

**Comment by ct-martin** [comment]
One more change, please move that giant chunk of CSS in the <head> to a separate file (styles.css) which should be linked last of the CSS files.

Also, @jflory7 - please double-check the copyright notice

**Comment by ct-martin** [comment]
Was not supposed to be closed, reopening after verifying with @axk4545

**Comment by axk4545** [comment]
@jflory7 if @jrtechs and @ct-martin say this is ready please review and merge. Then you or I can merge the website branch with master to get it into the right location while keeping master stable.

**Comment by axk4545** [comment]
@jrtechs @jflory7 @ct-martin where is this?

**Pull request closed without merging**